src/library/useUnsavedDirtyFormDataUtilities.js

```diff
@@ -19,10 +19,13 @@ export const useUnsavedDirtyFormDataUtilities = (sessionStorageName) => {
     [sessionStorageName],
   )
 
-  const getPersistedUnsavedFormData = useCallback(
-    () => JSON.parse(window.sessionStorage.getItem(sessionStorageName)),
-    [sessionStorageName],
-  )
+  const getPersistedUnsavedFormData = useCallback(() => {
+    try {
+      return JSON.parse(window.sessionStorage.getItem(sessionStorageName))
+    } catch {
+      return null
+    }
```

Comment:
dont know why, but this function was compalining about not being able to parse undefined. Felt like a good candidate for a try catch. (I think the error was dev server hotreloading, but whatever, it shouldnt kill the app for a hacky thing failing. Id rather the dirty form state be lost for a problem that im not sure actually exists anymore than the app crash.

Reply:
ohh interesting - yep good call adding a try/catch